src/coefficients/stdfunction_coefficient.cpp

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) 2019-2020, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+// # Author: Jonathan Wong @ LLNL.
```

Comment:
So I realized I technically said to do this in the contributing file. However in full disclosure I just copied it from BLT. I think a statement about "if you would like to denote authorship, feel free to do so. Otherwise it is tracked by GitHub." would suffice.